src/nav-soknad/styles/skjema.less

```diff
@@ -112,7 +120,8 @@
 
 .overskrift {
     margin-bottom: 1rem;
-    padding: 0 @skjemaMarg 0 @skjemaMarg;
+
```

Comment:
Superpirk: Hvorfor ny linje her? 🦸🏻‍♂️

Reply:
Ææ. Jeg har nok bare glemt å fjerne et whitespace. Jeg fikser. :)

Reply:
Haha, sykt viktig 😁

Reply:
Jeg er liksom vant til at Prettier fjerner sånt for meg. Men den er nok ikke like streng på .less filene våre 😅

Reply:
Fort gjort at man ønsker linjeskift her også for å gruppere ting sammen visuelt :)

Reply:
Den hadde sikkert fjernet om det var flere linjeskift (?)